source/components/defining-a-component.md

```diff
@@ -6,31 +6,70 @@ create a `components/blog-post` template.
 but `post` is not. This prevents clashes with current or future HTML element names, and
 ensures Ember picks up the components automatically.
 
-If you are including your Handlebars templates inside an HTML file via
-`<script>` tags, it would look like this:
-
-```handlebars
-<script type="text/x-handlebars" id="components/blog-post">
-  <h1>Blog Post</h1>
-  <p>Lorem ipsum dolor sit amet.</p>
-</script>
+A sample component template would look like this:
+
+```app/components/blog-post.hbs
+
+<h1>Blog Post</h1>
+<p>Lorem ipsum dolor sit amet.</p>
 ```
 
 If you're using build tools, create a Handlebars file at
-`templates/components/blog-post.handlebars`.
+`app/components/blog-post.hbs`.
 
 Having a template whose name starts with `components/` creates a
 component of the same name. Given the above template, you can now use the
 `{{blog-post}}` custom element:
 
-```handlebars
-<h1>My Blog</h1>
-{{#each post in model}}
-  {{blog-post}}
+<!--- <a class="jsbin-embed" href="http://jsbin.com/tikenoniku/1/edit?output">JS Bin</a><script src="http://static.jsbin.com/js/embed.js"></script> 
+
+The example above uses `<script>` tags to work inside of JSBin.--> 
+
+Ember-cli works by file structure, so there are no `<script>` tags:
+
+```app/templates/index.hbs
+
+{{#each}}
+  {{#blog-post title=title}}
+    {{body}}
+  {{/blog-post}}
 {{/each}}
 ```
 
-<a class="jsbin-embed" href="http://jsbin.com/juvic/embed?js,output">JS Bin</a><script src="http://static.jsbin.com/js/embed.js"></script>
+```app/templates/components/blog-post.hbs
+
+<article class="blog-post">
+  <h1>{{title}}</h1>
+  <p>{{yield}}</p>
+  <p>Edit title: {{input type="text" value=title}}</p>
+</article>
+```
+
+```app/routes/index.js
+
+import Ember from 'ember'
```

Comment:
Yes, we need to be consistent across all examples, and after discussing 
with @trek we are just going to leave the `import` out all together for 
now.